recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -25,451 +25,446 @@ class AwsSdkCppConan(ConanFile):
     # NON_GENERATED_CLIENT_LIST in src/cmake/sdks.cmake contains extra ones that should also be added here
     # and that's the list of sdks. Then join it to this one
     _sdks = (
```

Comment:
Removed support for multiple versions, this simplifies the maintenance when adding newer ones if we're only ever keeping one around